packages/ui/src/utils/field-validations.ts

```diff
@@ -22,7 +27,29 @@ export const usernameValidation = (username: string): ErrorType | undefined => {
   }
 
   if (!usernameRegex.test(username)) {
-    return 'username_valid_charset';
+    return 'username_invalid_charset';
+  }
+};
+
+export const emailValidation = (email: string): ErrorType | undefined => {
+  if (!emailRegex.test(email)) {
+    return 'invalid_email';
+  }
+};
+
+export const phoneValidation = (value: string): ErrorType | undefined => {
```

Comment:
Again, function name should be verb-object phrase. Thus, `validateEmail` and `validatePhone`